ArjunaJTS/orbportability/pom.xml

```diff
@@ -122,7 +122,6 @@
       <groupId>org.jboss.openjdk-orb</groupId>
       <artifactId>openjdk-orb</artifactId>
       <version>${version.org.jboss.openjdk-orb}</version>
-      <scope>test</scope>
```

Comment:
Are you able to share why `test` is no longer sufficient?

Reply:
No sure why I removed `test`. I will restore it.